backend/dataall/modules/datapipelines/cdk/pivot_role_datapipelines_policy.py

```diff
@@ -54,11 +54,11 @@ def get_statements(self):
                 ],
```

Comment:
I have always had my doubts about this cloudformation policy. At the moment it is needed because DDK does not enforce a naming convention on the stacks. Do you think we should work on restricting it?